moz/templates/documents_list.html

```diff
@@ -1,56 +1,34 @@
 {% extends 'layout.html' %} {% block body %}
 
-<div class="row marketing">
-    <div class="panel-group" id="accordion">
-        <div class="panel panel-default">
-            <div class="panel-heading">
-                <h4 class="panel-title">
-                    <a data-toggle="collapse" data-parent="#accordion" href="#collapse1">Category 1 <span class="badge pull-right">2</span></a>
-                </h4>
-            </div>
-            <div id="collapse1" class="panel-collapse collapse">
-                <ul class="list-group">
-                    <li class="list-group-item">One</li>
-                    <li class="list-group-item">Two</li>
-                    <li class="list-group-item">Three</li>
-                </ul>
-            </div>
-        </div>
-        <div class="panel panel-default">
-            <div class="panel-heading">
-                <h4 class="panel-title">
-                    <a data-toggle="collapse" data-parent="#accordion" href="#collapse2">Category 2 <span class="badge pull-right">5</span></a>
-                </h4>
-            </div>
-            <div id="collapse2" class="panel-collapse collapse">
-                <ul class="list-group">
-                    <li class="list-group-item">One</li>
-                    <li class="list-group-item">Two</li>
-                    <li class="list-group-item">Three</li>
-                </ul>
-            </div>
-        </div>
-        <div class="panel panel-default">
-            <div class="panel-heading">
-                <h4 class="panel-title">
-                    <a data-toggle="collapse" data-parent="#accordion" href="#collapse3">Category 3 <span class="badge pull-right">3</span></a>
-                </h4>
-            </div>
-            <div id="collapse3" class="panel-collapse collapse">
-                <ul class="list-group">
-                    <li class="list-group-item">One</li>
-                    <li class="list-group-item">Two</li>
-                    <li class="list-group-item">Three</li>
-                </ul>
-            </div>
+    <div class="row marketing">
+        <div class="panel-group" id="accordion">
+            {% for c in categories %}
+                <div class="panel panel-default">
+                    <div class="panel-heading">
+                        <h4 class="panel-title">
+                            <a data-toggle="collapse" data-parent="#accordion" href="#collapse{{ c.id }}"> {{ c.title }}
+                                <span
+                                        class="badge pull-right">{{ c.documents|length }}</span></a>
```

Comment:
I'm not sure, but what does the sign "|" mean here?

Reply:
Its like calling document.length